hugegraph-server/hugegraph-core/src/main/java/org/apache/hugegraph/backend/store/BackendStoreProvider.java

```diff
@@ -67,6 +67,12 @@ public interface BackendStoreProvider {
 
     void listen(EventListener listener);
 
+    default void listenSchemaCacheClear(EventListener listener) {
```

Comment:
prefer listenSchemaCacheAtMostOnce

Reply:
thanks ,  you mean to update the method name to  listenSchemaCacheAtMostOnce?